src/twisted/persisted/dirdbm.py

```diff
@@ -235,18 +258,20 @@ def get(self, key, default = None):
         else:
             return default
 
+
     def __contains__(self, key):
         """
         C{key in dirdbm}
 
         @type key: str
         @param key: The key to test
-                
+
```

Comment:
Should this just call has_key? This is literally copy/paste (and we had to fix the same issues twice)